SwiftyBeaverTests/BaseDestinationTests.swift

```diff
@@ -343,9 +343,14 @@ class BaseDestinationTests: XCTestCase {
             path: "/world/ViewController.swift",
             function: "myFunc",
             message: "Hello World"))
+        
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Verbose,
+            path: "/world/ViewController.swift",
+            function: "myFunc",
+            message: "Hello World"))
 
         // not in filter but matching global minLevel
-        XCTAssertTrue(destination.shouldLevelBeLogged(.Info,
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Info,
```

Comment:
Absolutely! Filters are relatively new and I just wanted it to settle down a bit until it gets documentation. Docs are coming soon now. Thanks for digging so deep into filters even with docs. That’s amazing and you earned now the SwiftyBeaver Expert badge 🏆 🎉

Feel free to point me in other directions where you think SwiftyBeaver needs improvement. And also let's use Slack. Thanks again!